src/bundle/Resources/views/admin/policy/edit.html.twig

```diff
@@ -1,35 +1,39 @@
-{% extends "@EzPlatformAdminUi/layout.html.twig" %}
+{% extends "@EzPlatformAdminUi/admin/policy/base.html.twig" %}
 
 {% trans_default_domain 'role' %}
 
-{% block breadcrumbs %}
+{% block breadcrumbsAdmin %}
     {% include '@EzPlatformAdminUi/parts/breadcrumbs.html.twig' with { items: [
         { value: 'breadcrumb.admin'|trans(domain='messages')|desc('Admin') },
         { url: path('ezplatform.role.list'), value: 'role.breadcrumb.list'|trans|desc('Roles') },
-        { url: path('ezplatform.role.view', {roleId: role.id}), value: 'role.breadcrumb.view'|trans({ '%identifier%': role.identifier })|desc('%identifier%') },
-        { value: 'policy.breadcrumb.edit'|trans({ '%module%': policy.module|capitalize, '%function%': policy.function|capitalize })|desc('Editing Policy "%module% / %function%"') }
+        { url: path('ezplatform.role.view', {roleId: role.id}), value: 'role.breadcrumb.view.identifier'|trans({ '%identifier%': role.identifier })|desc('Role "%identifier%"') },
+        { value: 'policy.breadcrumb.edit'|trans()|desc('Edit limitations') }
     ]} %}
 {% endblock %}
 
-{% block pageTitle %}
+{% block pageTitleAdmin %}
     {% include '@EzPlatformAdminUi/parts/page_title.html.twig' with {
-        title: 'policy.view.edit.title'|trans({ '%module%': policy.module|capitalize, '%function%': policy.function|capitalize })|desc('Editing Policy "%module% / %function%"'),
+        title: 'policy.view.edit.title'|trans()|desc('Edit limitations'),
         iconName: 'roles'
     } %}
 {% endblock %}
 
-{% block content %}
+{% block form %}
     {{ form_start(form) }}
     <section class="container mt-4 px-5">
-        <div class="card">
-            <div class="card-body">
-                {{ form_row(form.policy) }}
+        <div class="card ez-card">
+            <div class="ez-card__title">
+                <h5>{{ 'policy.view.policy.title'|trans|desc('Editing - Policy type') }}</h5>
+            </div>
+            <div class="ez-card__body">
+                {{ policy.module|capitalize ~ ' / ' ~ policy.function|capitalize }}
+                {% do form.policy.setRendered %}
             </div>
         </div>
 
-        <div class="card ez-adminsection-edit mt-4">
+        <div class="card ez-adminsection-edit mt-4 ez-card">
             <div class="card-header">
-                <h2>{{ 'policy.view.limitations.title'|trans|desc('Limitations') }}</h2>
+                <h2>{{ 'policy.view.limitations.title'|trans|desc('Edit limitations') }}</h2>
```

Comment:
`.ez-card__title`

Reply:
I changed .ez-card__title to .ez-card__header and it is not used any more